airflow.cfg

```diff
@@ -216,7 +216,7 @@ daemon_umask = 0o077
 # SqlAlchemy supports many different database engines.
 # More information here:
 # http://airflow.apache.org/docs/apache-airflow/stable/howto/set-up-database.html#database-uri
-sql_alchemy_conn = sqlite:////opt/airflow/airflow.db
+sql_alchemy_conn = postgresql+psycopg2://airflow:airflow@postgres/airflow
```

Comment:
do we need a secure/obfuscated pg password in production? my thinking was that if an unintended user can reach the postgres instance we have bigger problems.